lte/gateway/docker/ghz/Dockerfile

```diff
@@ -0,0 +1,52 @@
+# AGW test docker private repository location
```

Comment:
Do we need a copyright header here?

Reply:
None of our other Dockerfiles have copyright header added, maybe this can be added on a separate PR but that's why I left it like that